src/core/grpc_service.proto

```diff
@@ -426,6 +448,13 @@ message SharedMemoryControlResponse
   //@@     The status of the request, indicating success or failure.
   //@@
   RequestStatus request_status = 1;
+
+  //@@
+  //@@  .. cpp:var:: SharedMemoryRegion shared_memory_region
+  //@@
+  //@@     The list of active/registered shared memory regions.
+  //@@
+  repeated InferSharedMemoryRegion shared_memory_region = 2;
```

Comment:
This is only relevant for "Status" so again so a oneof with a "Status" message where in that message is this repeated. Also status results should be returned as a SharedMemoryStatus message, right?

Reply:
That is correct

Reply:
So this needs to be a oneof which (currently) just has a "Status" message which includes SharedMemoryStatus.